src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -3348,6 +3446,76 @@ export function geneFilterQueryFromOql(
     };
 }
 
+export function structVarFilterQueryFromOql(
```

Comment:
seems odd that we wouldn't reuse parsing from general oql handling for query functionality

Reply:
Ok, this is possible but not straight forward. I had to convert the gene1 and gene2 values to a parsed OQL object and then convert it to a OQL string using the unparseOQLQueryLine function in _oqlfilter.ts_.

```
    const gene1 = query.gene1Query.hugoSymbol || '';
    const gene2 = query.gene2Query.hugoSymbol || '';
    // Translate to OQL SingleGeneQuery object.
    const parsed_oql_line: SingleGeneQuery = gene1 ?
        { gene: gene1, alterations: [{ gene: gene2, alteration_type: STUCTVARDownstreamFusionStr, modifiers: [] }]}:
        { gene: gene2, alterations: [{ gene: gene1, alteration_type: STUCTVARUpstreamFusionStr, modifiers: [] }]};
    return unparseOQLQueryLine(parsed_oql_line);
```

Do think this is better?